ledger-core-bitcoin/src/CMakeLists.txt

```diff
@@ -96,6 +96,10 @@ else()
     include(StaticAnalyzers)
     set_enabled_static_analyzers(ledger-core-bitcoin-obj)
 
+    # add compiler warnings
+    include(CompilerWarnings)
+    set_enabled_compiler_warnings(ledger-core-bitcoin-obj)
```

Comment:
Argh, I didn’t see those names, `set_enabled`, what does it mean? It’s a bit confusing… :confused: